hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -487,6 +487,22 @@
           tags: [// This CSR only briefly pulses to anything other than 0.
                  "excl:CsrNonInitTests:CsrExclWrite"]
         },
+        { bits: "1",
+          name: "FLASH_READ_BUFFER_CLR",
+          swaccess: "rw1s",
```

Comment:
is this a single-cycle operation or do we need the ability to poll clearing status?
if it is the former, it may make sense to make this a `swaccess: wo` and `hwaccess: hro` register.

Reply:
It's a single cycle, the same as the field above. wo / hro works for me, and I wasn't sure which way best represented the intention. Just need to add the qe in that case.

Reply:
Ah I should have expanded that in the viewer :). 
wo/hro seems more concise, but this is likely incompatible with the higher `MODE` bits present.
So in order to avoid further changes to the CSR layout, I think it's ok to merge this as-is.